README.md

```diff
@@ -30,29 +28,60 @@ poetry add supabase_auth
 pip install supabase_auth
 ```
 
+## Features
+
+- Full feature parity with the JavaScript client
+- Support for both synchronous and asynchronous operations
+- MFA (Multi-Factor Authentication) support
+- OAuth and SSO integration
+- Magic link and OTP authentication
+- Phone number authentication
+- Anonymous sign-in
+- Session management with auto-refresh
+- JWT token handling and verification
+- User management and profile updates
+
 ## Differences to the JS client
 
-It should be noted there are differences to the [JS client](https://github.com/supabase/gotrue-js). If you feel particulaly strongly about them and want to motivate a change, feel free to make a GitHub issue and we can discuss it there.
+It should be noted there are differences to the
```

Comment:
I'm not sure what is going on here but the content is getting broken into multiple lines instead of a single line.